views/settings-page.twig

```diff
@@ -401,6 +401,17 @@
                                     {% endif %}
                                 {% endif %}
 
+                                {% if theme.isSettingVisible("QUICK_CHART_URL") %}
+                                    {% set title %}{% trans "Quick Chart URL" %}{% endset %}
+                                    {% set helpText %}{% trans "Quick Chart URL for reporting and 2FA." %}{% endset %}
```

Comment:
Perhaps:

"Enter the URL to a Quick Chart service. This is used to draw charts in emailed reports and for showing a QR code during two factor authentication."